### PR TITLE
Packaging improvements

### PR DESCRIPTION
**Pull request by rohanpm**
- Ensure supporting files are included in distribution (most critically, LICENSE)
- Clean up unnecessary code
- Make setup.py read from requirements.txt
- Clarify license

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,11 +34,5 @@ def get_long_description():
...
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
-    install_requires=[
-        'requests',
-        'more-executors',
-        'six',
-        'monotonic',
-        'edgegrid-python',
-    ],
+    install_requires=open('requirements.txt').readlines(),
```
Comment (danrodrig) [inline setup.py]: Non blocking comment, I would go with open().read().splitlines() to avoid the new line characters.

File: setup.py [hunk 2]
```diff
@@ -6,13 +6,7 @@ def get_description():
...
 
 def get_long_description():
-    try:
-        text = open('README.md').read()
-    except IOError as error:
-        if error.errno == 2:
-            # fallback for older setuptools
-            return get_description()
-        raise
+    text = open('README.md').read()
```
Comment (alexandrevicenzi) [inline setup.py]: Closing files is always good.

```python
with open('README.md') as f:
     text = f.read()
```

File: setup.py (hunk 1 above)
Comment (rohanpm) [reply setup.py]: Fixed.

File: setup.py (hunk 2 above)
Comment (rohanpm) [reply setup.py]: Fixed.

**Comment by rohanpm** [comment]
There's a problem with coveralls all of a sudden, affecting all pull requests.  Please ignore it here.  I've filed separate issue https://github.com/release-engineering/python-fastpurge/issues/12 for that.

**Pull request merged**